react-native-libraries.json

```diff
@@ -6752,5 +6758,18 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/akinncar/react-native-mask-text",
+    "examples": [
+      "https://github.com/akinncar/react-native-mask-text/tree/main/example",
+      "https://snack.expo.io/@akinncar/react-native-mask-text"
+    ],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expo": true,
+    "windows": true,
+    "macos": true
```

Comment:
To mark out-of-tree platforms as supported, the library examples had to include those platforms. The `web` platform is threaten a bit differently because of build-in support for this platform in Expo.

This requirements has been added due to compatibility issues reported by users of some libraries in the past.

Reply:
got it! thank's for your reply, I will provide the examples on the library for windows and mac os asap and will make another PR when it's done.
in the meantime, I removed the support indication for these platforms.